mercury/templates/dashboard.html

```diff
@@ -60,8 +60,9 @@ <h3>The Simulated Data for each snapshot</h3>
 	<div class="row">
         <div id="articles" style="width:1400px;height:800px;"></div>
     </div>
-	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
-	<script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+{#	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>#}
```

Comment:
```suggestion
```

Reply:
I would just remove this commented line.

Reply:
okay